scripts/chandra/chandra.py

```diff
@@ -0,0 +1,209 @@
+import h5py
+import numpy as np
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@ARTICLE{2010ApJS..189...37E,
+       author = {{Evans}, Ian N. and {Primini}, Francis A. and {Glotfelty}, Kenny J. and {Anderson}, Craig S. and {Bonaventura}, Nina R. and {Chen}, Judy C. and {Davis}, John E. and {Doe}, Stephen M. and {Evans}, Janet D. and {Fabbiano}, Giuseppina and {Galle}, Elizabeth C. and {Gibbs}, Danny G., II and {Grier}, John D. and {Hain}, Roger M. and {Hall}, Diane M. and {Harbo}, Peter N. and {He}, Xiangqun Helen and {Houck}, John C. and {Karovska}, Margarita and {Kashyap}, Vinay L. and {Lauer}, Jennifer and {McCollough}, Michael L. and {McDowell}, Jonathan C. and {Miller}, Joseph B. and {Mitschang}, Arik W. and {Morgan}, Douglas L. and {Mossman}, Amy E. and {Nichols}, Joy S. and {Nowak}, Michael A. and {Plummer}, David A. and {Refsdal}, Brian L. and {Rots}, Arnold H. and {Siemiginowska}, Aneta and {Sundheim}, Beth A. and {Tibbetts}, Michael S. and {Van Stone}, David W. and {Winkelman}, Sherry L. and {Zografou}, Panagoula},
+        title = "{The Chandra Source Catalog}",
+      journal = {\apjs},
+     keywords = {catalogs, X-rays: general, Astrophysics - High Energy Astrophysical Phenomena, Astrophysics - Instrumentation and Methods for Astrophysics},
+         year = 2010,
+        month = jul,
+       volume = {189},
+       number = {1},
+        pages = {37-82},
+          doi = {10.1088/0067-0049/189/1/37},
+archivePrefix = {arXiv},
+       eprint = {1005.4665},
+ primaryClass = {astro-ph.HE},
+       adsurl = {https://ui.adsabs.harvard.edu/abs/2010ApJS..189...37E},
+      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+Spectra from the Chandra Source Catalog. Processed from pulse height
+amplitude (PHA) files modified by instrumental response files ARF and RMF.
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://cxc.cfa.harvard.edu/csc/"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = ""
+
+_VERSION = "2.1.0"
+
+# Full data model here:
+# https://desidatamodel.readthedocs.io/en/latest/DESI_SPECTRO_REDUX/SPECPROD/zcatalog/zpix-SURVEY-PROGRAM.html
+
+#_BOOL_FEATURES = [
+#    "ZWARN"
+#]
+
+_FLOAT_FEATURES = [
+    "flux_aper_b",
+    "flux_bb_aper_b",
+    "flux_significance_b",
+    "hard_hm",
+    "hard_hs",
+    "hard_ms",
+    "var_index_b",
+    "var_prob_b",
+    
+]
+
+class CHANDRA(datasets.GeneratorBasedBuilder):
+    """TODO: Short description of my dataset."""
+
+    VERSION = _VERSION
+    
+    
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="csc21_spectra",
+            version=VERSION,
+            data_files=DataFilesPatternsDict.from_patterns(
+                {"train": ["./output_files/parent*.hdf5"]}
```

Comment:
```suggestion
                {"train": ["./output_data/parent*.hdf5"]}
```
Matching the default in the args of the other files.